### ISSUE TITLE
[A11y] Calendar picker UI is not accessible through keyboard navigation

### ISSUE THREAD

**Issue by michaelktiufb** [hlxsites/accenture-newsroom#421]
1. Navigate to https://tempnewsroom.accenture.com/
2. Navigate to the calendar picker icon through tabbing
3. Open the calendar picker modal
4. Click the tab key

ER The focus is on the elements within the calendar modal
AR The calendar modal elements were skipped and the focus directly wen to the elements beneath it

**Comment by amol-anand** [comment]
This works exactly like in the current newsroom.accenture.com site. We are also using the same datepicker library.

https://github.com/hlxsites/accenture-newsroom/assets/7074782/bcd674d8-12f5-4a68-94e5-41beaf5108ce
